src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy

```diff
@@ -38,8 +41,9 @@ final class ACESeqMethods {
                 ACEseqConstants.TOOL_CNV_SNP_GENERATION,
                 tumorBam,
                 controlBam,
-                ACEseqConstants.PARM_CHR_INDEX,
-                getGlobalJobSpecificParameters(controlBam.executionContext.configuration))
+                ACEseqConstants.PARM_CHR_INDEX)
+//        ,
```

Comment:
Remove comment.

Reply:
Nope. The code needs to stay but I need the proper ParallelizationHelper first. Won't compile otherwise.

Reply:
Copied code from the other COBasePlugin Repo. See issue https://github.com/eilslabs/COWorkflowsBasePlugin/commit/d5caf42507448182e55692b81f173b459fe1844e.